app/controllers/subscriptions_controller.rb

```diff
@@ -7,7 +7,8 @@ class SubscriptionsController < ApplicationController
   def new
     @upgrade_user = params[:user_id]
     @sponsorship = @upgrade_user && current_user.try(:id) != @upgrade_user
-    render plan_name
+    plan = params[:plan] || 'premium'
```

Comment:
Rather than hardcoding your 'default' plan, you could indicate which plan is default by a flag in the new Plans table.  Then you could change the terms of the default plan for new people while keeping existing people on the plan they came in on.